modules/nlu/src/backend/application/training-queue.ts

```diff
@@ -3,7 +3,7 @@ import _ from 'lodash'
 
 import moment from 'moment'
 import ms from 'ms'
-import nanoid from 'nanoid'
+import { nanoid } from 'nanoid'
```

Comment:
Can you please also install typings for this library (or you can also upgrade it to >=V3 which contains its own typings.

Also, when I go to the node_modules of the NLU module, there only seem to be a default export:

![Screenshot from 2021-09-02 18-12-49](https://user-images.githubusercontent.com/9640576/131923173-85ddaa6b-879f-4521-b77a-8f76f447452b.png)

I have quickly read the issue you linked and I think upgrading might be the easiest solution. What do you think?

Reply:
To add more info, I get these errors when trying to build the module with the typings installed (`@types/nanoid": "^2.0.1`):

![Screenshot from 2021-09-02 18-16-49](https://user-images.githubusercontent.com/9640576/131923415-30485346-fbec-4d1e-8149-a83bd6cd49e5.png)